recipe/meta.yaml

```diff
@@ -28,7 +28,7 @@ requirements:
     - pip
```

Comment:
is make actually needed ?, otherwise LGTM

Reply:
Actually, not sure why compiler is needed too. This is a rebuild so I did not pay attention to that.